docker-env.example

```diff
@@ -24,4 +24,6 @@ NUMBER_OF_SYNC_LOG=5
 # Shell with unicode support
 LANG=C.UTF-8
 
```

Comment:
@radinreth please add a comment line with a link to the documentation of Recaptcha